packages/mobile/lib/routers/helper.ts

```diff
@@ -93,5 +101,9 @@ function resetSubrouters(): void {
     get(profileRecoveryRouter).reset()
     get(profileSetupRouter).reset()
     get(storageProtectionSetupRouter).reset()
+}
+
+function resetWalletSubrouters(): void {
```

Comment:
based on the new convention, we can call this `dashboardSubrouters` instead of `wallet`